### ISSUE TITLE
BOUNDARY Hints in Markup

### ISSUE THREAD

**Issue by WardCunningham** [clemsos/wiki-plugin-map#12]
The BOUNDARY keyword could be followed by coordinates that should be included in the map but that do not define markers and are not communicated to other maps. The workflow for positioning a new map would then be:

- create a new map, possibly by dragging from the gallery
- position the map as desired
- shift-double-click to mark the location
- optionally, edit the markup to move the coordinate from the click marker

**Comment by WardCunningham** [comment]
I'm working on this now.

**Comment by WardCunningham** [comment]
Requires only a few lines of code but many tests to get right
```
    markup
      ✓ should accept caption only
      ✓ should accept marker only
      ✓ should accept mixed markers and caption
      ✓ should merge markers with lineup
      ✓ should separate markers from lineup for boundary
      ✓ should accept boundary without marker
      ✓ should accept multiple boundary without marker
      ✓ should add markers to boundary until stopped
```

**Comment by WardCunningham** [comment]
Implemented in #8.
